cocos/2d/CCLabel.cpp

```diff
@@ -525,6 +525,25 @@ void Label::reset()
     setRotationSkewX(0);        // reverse italics
 }
 
+//  ETC1 ALPHA supports, for LabelType::BMFONT & LabelType::CHARMAP
+static Texture2D* _getTexture(Label* label)
+ {
```

Comment:
why make a inner class in the function and cast the base class pointer as the derived class pointer...
Why not just add a new function in FontAtlas?